### PR TITLE
HTS-577: Implemented Deskpro Feedback functionality into our views

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -181,7 +181,7 @@ hts.helpers.sidebar.government-gateway=Government Gateway
 hts.helpers.sidebar.universal-credit=Universal Credit
 hts.helpers.sidebar.apply-for-universal-credit=Apply for Universal Credit
 hts.helpers.sidebar.working-tax-credit=Working Tax Credit
-hts.helpers.phaseBanner.feedback=This is a new service – your feedback will help us to improve it.
+hts.helpers.phaseBanner.feedback=This is a new service – your <a href="{0}">feedback</a> will help us to improve it.
```
Comment (AndyHWChung) [inline conf/messages]: Do we want HTML in our messages file? It hides it from the view file where the reset of the HTML is

Comment (Jackie1988) [reply conf/messages]: We don't need that anyway, so I have removed that along with the phaseBanner helper.

File: app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html [hunk 2]
```diff
@@ -103,17 +102,23 @@
...
 
-
 @getHelpForm = @{uiHelpers.reportAProblemLink(appConfig.reportAProblemPartialUrl, appConfig.reportAProblemNonJSUrl)}
 
+@betaBanner = {
+@uiLayouts.betaBanner(false, appConfig.betaFeedbackUrlNoAuth, appConfig.betaFeedbackUrlNoAuth, true, true)
+}
+
+@serviceInfoContent = {
+@uiLayouts.serviceInfo(betaBanner, false, Option.empty, true )
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html]: is `Option.empty` not just `None`?

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**